check/pytest

```diff
@@ -4,9 +4,11 @@
 # Runs pytest on the repository.
 #
 # Usage:
-#     check/pytest [--flags] [file-paths-relative-to-repo-root]
+#     check/pytest [--actually_quiet] [--flags for pytest] [file-paths-relative-to-repo-root]
```

Comment:
The flag should probably have a dash, not an underscore to be consistent with other flags.  `--actually-quiet`.

Reply:
Done.